pkg/virtctl/unpause/unpause_test.go

```diff
@@ -46,17 +45,19 @@ var _ = Describe("Unpausing", func() {
 	})
 
 	DescribeTable("should unpause VMI", func(unpauseOptions *v1.UnpauseOptions) {
-
-		vmi := api.NewMinimalVMI(vmName)
+		vmi := libvmi.New(
+			libvmi.WithNamespace(k8smetav1.NamespaceDefault),
```

Comment:
Same

Reply:
I tested it once again, without modifying the code.
I ran the command ginkgo kubevirt/pkg/virtctl/unpause
then got the long error:

.........
Ran 5 of 5 Specs in 0.007 seconds
FAIL! -- 1 Passed | 4 Failed | 0 Pending | 0 Skipped
--- FAIL: TestPauseUnpause (0.01s)
FAIL

Ginkgo ran 1 suite in 2.402377683s

Test Suite Failed